### PR TITLE
Add Create method to IDynamicContentRepositoryConductor

### PR DESCRIPTION
**Pull request by cspath1**
Addresses https://github.com/AndcultureCode/AndcultureCode.CSharp.Sitefinity.Core/issues/7

- [x] Related GitHub issue(s) linked in PR description
- [x] Destination branch merged, built and tested with your changes
- [x] Code formatted and follows best practices and patterns
- [x] Code builds cleanly (no _additional_ warnings or errors)
- [x] Manually tested
- [x] Automated tests are passing
- [x] No _decreases_ in automated test coverage
- [x] Documentation updated (readme, docs, comments, etc.)

### CODE REVIEW

File: src/AndcultureCode.CSharp.Sitefinity.Core/Interfaces/IDynamicContentRepositoryConductor.cs [hunk 1]
```diff
@@ -9,6 +10,14 @@ namespace AndcultureCode.CSharp.Sitefinity.Core.Interfaces
 {
     public interface IDynamicContentRepositoryConductor
     {
+        /// <summary>
+        /// Creates a Dynamic Content of the specified type T.
```
Comment (Stefanie899) [inline src/AndcultureCode.CSharp.Sitefinity.Core/Interfaces/IDynamicContentRepositoryConductor.cs]: Nitpick: Maybe `Creates a Dynamic Content item of the specified type T` or `Creates Dynamic Content of the specified type T`?

File: src/AndcultureCode.CSharp.Sitefinity.Core/Interfaces/IDynamicContentRepositoryConductor.cs [hunk 2]
```diff
@@ -9,6 +10,14 @@ namespace AndcultureCode.CSharp.Sitefinity.Core.Interfaces
...
+        /// </summary>
+        /// <typeparam name="T">A subclass of SitefinityContent.</typeparam>
+        /// <param name="item">The item.</param>
```
Comment (Stefanie899) [inline src/AndcultureCode.CSharp.Sitefinity.Core/Interfaces/IDynamicContentRepositoryConductor.cs]: Minor: The description of this param doesn't have any value. Suggest removing or expanding on what that property is for.

**Review by Stefanie899 — APPROVED** [review]

File: src/AndcultureCode.CSharp.Sitefinity.Core/Interfaces/IDynamicContentRepositoryConductor.cs (hunk 1 above)
Comment (KevinBusch) [inline src/AndcultureCode.CSharp.Sitefinity.Core/Interfaces/IDynamicContentRepositoryConductor.cs]: Suggest reworking to something like `Creates dynamic content of the specified type`.  The proper casing around `Dynamic Content` implies makes it appear to be an object which I think should be more generically stated in the description.

**Review by KevinBusch — APPROVED** [review]

**Comment by brandongregoryscott** [comment]
@all-contributors add @Stefanie899 for review

**Pull request merged**